panda.py

```diff
@@ -0,0 +1,224 @@
+from pxr import Usd, Vt
+import json
+
+data = {}
```

Comment:
This looks good for stage 1, but later on, we should make it a bit more generalizable.  We will want to likely make a script that uses the `argparse` library (a thing that allows us to use `--flags`) when calling it.  Think now about how this will look:
eg: `python usd_to_araas.py --file panda.usd --output panda_rigging.json`

When we create a program like this, we should structure it such that there is a main method that can either call some static functions or instantiates a generic class ready to go for parsing and exporting the robot kinematic structure.